cms/wagtail_api/schema/views.py

```diff
@@ -0,0 +1,93 @@
+"""
+Views for Wagtail API Schema
+"""
+
+from drf_spectacular.utils import OpenApiParameter, extend_schema
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .serializers import (
+    CertificatePageListSerializer,
+    CoursePageListSerializer,
+    PageListSerializer,
+)
+
+
+class WagtailPagesSchemaView(APIView):
```

Comment:
One (hopefully) minor thing: this doesn't expose a retrieve view in the spec. We may not need it necessarily for the certificates APIs (see review comment) but we will for other things.